integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -137,15 +138,38 @@ def _transmit_action(self, content_metadata_item_map, client_method, action_name
             try:
                 response_status_code, response_body = client_method(serialized_chunk)
             except ClientError as exc:
+                LOGGER.exception(exc)
+                response_status_code = exc.status_code
+                response_body = exc.message
                 self._log_error(
                     f"Failed to {action_name} [{len(chunk)}] content metadata items for integrated channel "
                     f"[{self.enterprise_configuration.enterprise_customer.name}] "
                     f"[{self.enterprise_configuration.channel_code()}]. "
-                    f"Task failed with message [{exc.message}] and status code [{exc.status_code}]"
+                    f"Task failed with message [{exc.message}] and status code [{response_status_code}]"
                 )
+            except requests.exceptions.RequestException as exc:
```

Comment:
lets catch more error states for thoroughness but also to free up clients from having to implement a lot of error handling